src/actions/preferences.js

```diff
@@ -13,12 +13,12 @@ import {getPreferenceKey} from 'utils/preference_utils';
 import {bindClientFunc, forceLogoutIfNecessary} from './helpers';
 import {getLogErrorAction} from './errors';
 
-export function deletePreferences(preferences) {
+export function deletePreferences(userId, preferences) {
```

Comment:
Right but you're passing in an array of preferences, I find it odd to pick one of the preferences in the array at random to use the user id for. I think it's much better to just be explicit